src/test/java/persistence/sql/meta/ColumnFieldTest.java

```diff
@@ -0,0 +1,84 @@
+package persistence.sql.meta;
+
+import jakarta.persistence.*;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class ColumnFieldTest {
+
+    @Entity
+    public class DummyPerson {
+
+        @Id
+        @GeneratedValue(strategy = GenerationType.IDENTITY)
+        private Long id;
+
+        @Column(name = "nick_name")
+        private String name;
+
+        @Column(name = "old")
+        private Integer age;
+
+        @Column(nullable = false)
+        private String email;
+
+    }
+
+    @Entity
+    public class DummyPerson2 {
+
+        @Id
+        @GeneratedValue(strategy = GenerationType.IDENTITY)
+        private Long id;
+
+        @Id
+        @Column(name = "nick_name")
+        private String name;
+
+        @Column(name = "old")
+        private Integer age;
+
+        @Column(nullable = false)
+        private String email;
+
+        @Transient
+        private Integer index;
+
+    }
+
+    @Test
+    @DisplayName("임의의 Class에 정의된 Field 이름 가져오기")
+    void getColumnFieldNames() {
+        List<String> expected = Arrays.asList("id", "nick_name", "old", "email");
+        assertThat(new ColumnFields(DummyPerson.class).getDeclaredFieldNames())
+                .isEqualTo(expected);
+    }
+
+    @Test
+    @DisplayName("임의의 Transient Annotation이 포함된 Class에 정의된 Field 이름 가져오기")
+    void getColumnFieldNames_Transient() {
+        List<String> expected = Arrays.asList("id", "nick_name", "old", "email");
+        assertThat(new ColumnFields(DummyPerson2.class).getDeclaredFieldNames())
+                .isEqualTo(expected);
+    }
+
+    @Test
+    @DisplayName("임의의 Class의 Primary Column 가져오기")
+    void getPrimary() {
+        List<String> expected = Arrays.asList("id");
+        assertThat(new ColumnFields(DummyPerson.class).getPrimary().stream().map(ColumnField::getName)).isEqualTo(expected);
+    }
+
+    @Test
+    @DisplayName("임의의 Class의 Primary Column 가져오기 - id가 여러개 정의됨")
+    void getPrimary_multiple() {
+        List<String> expected = Arrays.asList("id", "nick_name");
+        assertThat(new ColumnFields(DummyPerson2.class).getPrimary().stream().map(ColumnField::getName)).isEqualTo(expected);
+    }
```

Comment:
테스트도 가독성을 위해 bdd(given-when-then)을 이용해보시는건 어떨까요?
```suggestion
    @Test
    @DisplayName("임의의 Class의 Primary Column 가져오기 - id가 여러개 정의됨")
    void getPrimary_multiple() {
        // given
        final ColumnFields columnFields = new ColumnFields(DummyPerson2.class);
        // when
        final List<String> actual = columnFields.getPrimary().stream().map(ColumnField::getName).toList();
        // then
        assertThat(actual).containsExactly("id", "nick_name");
    }
```
개인적으로 여기서 `columnFields.getPrimaryNames` 와 같은 메서드를 추가해 객체에게 메시지를 전달하는것을 지향합니다 :)